FinSearchUnified/Helper/StaticHelper.php

```diff
@@ -823,4 +823,30 @@ public static function setPromotion(SimpleXMLElement $xmlResponse)
             ]);
         }
     }
+
+    /**
+     * @param \SimpleXMLElement $xmlResponse
+     */
+    public static function setSmartDidYouMean(SimpleXMLElement $xmlResponse)
+    {
+        $query = $xmlResponse->query;
+        $originalQuery = (string)$query->originalQuery;
+        $didYouMeanQuery = (string)$query->didYouMeanQuery;
+        $queryString = $query->queryString;
+        $queryStringType = (string)$queryString->attributes()->type;
+
+        if (((isset($originalQuery) && $originalQuery !== '') || (isset($didYouMeanQuery) && $didYouMeanQuery !== ''))
```

Comment:
Instead of `(isset($originalQuery) && $originalQuery !== '')` use `!empty($originalQuery)`. Should yield the same result.